stl/inc/xmemory

```diff
@@ -1796,6 +1802,12 @@ void uninitialized_fill(const _NoThrowFwdIt _First, const _NoThrowFwdIt _Last, c
     if constexpr (_Fill_memset_is_safe<_Unwrapped_t<const _NoThrowFwdIt&>, _Tval>) {
         _Fill_memset(_UFirst, _Val, static_cast<size_t>(_ULast - _UFirst));
     } else {
+        if constexpr (_Fill_zero_memset_is_safe<_Unwrapped_t<const _NoThrowFwdIt&>, _Tval>) {
+            if (_Is_scalar_zero(_Val)) {

```

Comment:
Ditto "Does this new special case have test coverage?"

Reply:
`VSO_0180469_fill_family` already tests for 0